packages/asc-ui/src/utils/themeUtils.ts

```diff
@@ -152,6 +152,22 @@ export const focusStyleText = () => ({
   }
 `
 
+export enum FocusStyleEnum {
+  outline,
+  fill,
+  none,
+}
+
+export const getFocusStyle = (focusStyle: string = 'fill') => {
```

Comment:
wrong type, should be: focusStyle: keyof typeof FocusStyleEnum

Reply:
ok